go/cmd/vtorc/cli/cli.go

```diff
@@ -38,22 +32,11 @@ var (
 	configFile string
 	Main       = &cobra.Command{
 		Use:     "vtorc",
-		Short:   "", // TODO
+		Short:   "VTOrc is the automated fault detection and repair tool of Vitess.",
```

Comment:
```suggestion
		Short:   "VTOrc is the automated fault detection and repair tool in Vitess.",
```

Reply:
happy to make this change but i copied this verbatim from https://vitess.io/docs/18.0/reference/vtorc/architecture/. should we change that too?

Reply:
Yes please. Let's change that as well.